test/sqllogictest/select_introspection.slt

```diff
@@ -41,35 +41,6 @@ a b
 statement ok
 SELECT * FROM mz_materializations
 
-# Verify that we are always hitting the same replica.
```

Comment:
This test is always breaking and I can never tell why, it's already extremely flaky and we do a bunch of hacky things to get it to pass. I just decided to delete it once and for all. If you feel strongly about not deleting it, then I will spend the time to try and fix it again.

Reply:
Yes, this test is a nightmare to maintain. I'm OK with removing it.